#!/usr/bin/python3

def raise_exception_msg(message=""):
	"""just add a message this time"""
	raise NameError(message)
